collectors/base/config.py

```diff
@@ -66,3 +66,7 @@
 HRA_URL = os.environ.get('HRA_URL', None)
 HRA_USER = os.environ.get('HRA_USER', None)
 HRA_PASS = os.environ.get('HRA_PASS', None)
+
+# Cochrane Reviews
+
+COCHRANE_ARCHIVE_URL = os.environ.get('COCHRANE_ARCHIVE_URL', None)
```

Comment:
I know that this file is using this pattern of `foo.get('BAR', None)`. However, this is the same as `foo.get('BAR')`, so that would be preferred.

I want to go over the entire codebase removing this pattern later, but haven't got the time to do it yet.